doc/nrf/libraries/others/emds.rst

```diff
@@ -9,45 +9,47 @@ Emergency data storage
 
 Overview
 ********
-The emergency data storage (EMDS) library provides persistent storage functionality designed to prevent the wear and tear of the flash memory.
+The emergency data storage (EMDS) library provides persistent storage functionality designed to prevent the wear and tear of the flash memory or RRAM.
 Its intended use is for storing of data undergoing frequent changes during runtime.
 
 Implementation
 ==============
 The :kconfig:option:`CONFIG_EMDS` Kconfig option enables the emergency data storage.
 
-The application must initialize the pre-allocated flash area by using the :c:func:`emds_init` function.
+The application must initialize the pre-allocated storage area by using the :c:func:`emds_init` function.
 The :kconfig:option:`CONFIG_EMDS_SECTOR_COUNT` option defines how many sectors should be used to store data.
 
 The allocated storage space must be larger than the combined data stored by the application.
-Allocating a larger flash area will demand more resources, but also increase the life expectancy of the flash.
-The chosen size should reflect the amount of data stored, the available flash resources, and how the application calls the :c:func:`emds_store` function.
-In general, it should not be necessary to allocate a large flash area, since only a limited set of data should be stored to ensure swift completion of writing the flash on shutdown.
+Allocating a larger storage area will demand more resources, but also increase the life expectancy of the flash/RRAM.
```

Comment:
My idea was to replace the remaining "flash/RRAM" with "flash or RRAM", I'm not really a fan of the slashes. Up to you, won't block on this.